### PR TITLE
Spanish edition of TW5, based on the french one

### PR DESCRIPTION
**Pull request by PauDeLuca**
-Working on a spanish version of tiddlywiki, based on @xcazin 's french edition.
-Updated es-ES plugin translation, extracted from the submitted hmtl-file:
https://dl.dropboxusercontent.com/u/885342/Translation_ES-es_5.1.12-prerelease.html

### CODE REVIEW

**Comment by Jermolene** [comment]
Hi @PauDeLuca thank you, that's great.

It looks like there's some modifications to files in the `fr-FR` edition, is that intentional?

**Comment by PauDeLuca** [comment]
Did I make changes in the fr-FR folders? If yes, it's not intentional at all. Please, forgive my clumsiness.

The only files I've intentionally modified are in the es-ES and es-ES-server folders. In them, I've modified three of the original files in the core/UI (ViewTemplate, ListItemTemplate and DefaultSearchResult) so they don't longer need the field fr-title, and accept the field es-title, so the original tiddler names in english don't need to be modified, nor the system's cross-references.

**Comment by Jermolene** [comment]
Hi @PaulDeLuca no worries, GitHub is tricksy and this sort of thing happens quite often.

You need to remove the extraneous changes from the pull request; the easiest way is to copy the original files into your working directory, and then commit to the pull request. Of course, as ever with git, if the worst comes to the worst you may find it easier to recreate the pull request.

**Comment by PauDeLuca** [comment]
I've restored the original files to fr-FR and fr-FR-server.
I managed to tangle my feet with my own commits, but I hope it's now all right, allthough I can't see my push. Please confirm everything is ok now.

**Comment by PauDeLuca** [comment]
Now I see my pushes. Please confirm that fr-FR is now okay... and sorry again.

File: editions/es-ES/tiddlers/system/download-empty.tid [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title: $:/editions/es-ES/download-empty
+type: text/vnd.tiddlywiki
+
+\define saveTiddlerFilter()
+[[$:/core]] [[$:/isEncrypted]] [[$:/themes/tiddlywiki/snowwhite]] [[$:/themes/tiddlywiki/vanilla]] [[$:/languages/fr-FR]] [[$:/language]] -[[$:/boot/boot.css]] -[type[application/javascript]library[yes]] -[[$:/boot/boot.js]] -[[$:/boot/bootprefix.js]] +[sort[title]]
```
Comment (Jermolene) [inline editions/es-ES/tiddlers/system/download-empty.tid]: I think `fr-FR` should be `es-ES` here

File: editions/fr-FR/tiddlers/$__StoryList.tid [hunk 2]
```diff
@@ -0,0 +1,6 @@
+created: 20160411220313489
+list: $:/AdvancedSearch $:/core/ui/DefaultSearchResultList $:/core/ui/ViewTemplate/title $:/core/ui/ListItemTemplate HelloThere GettingStarted Community
+modified: 20160411220313489
+title: $:/StoryList
```
Comment (Jermolene) [inline editions/fr-FR/tiddlers/$__StoryList.tid]: This appears to be the one remaining tiddler from `editions/fr-FR`

File: editions/es-ES/tiddlers/es-ES/ControlPanel.multids [hunk 3]
```diff
@@ -0,0 +1,141 @@
+title: $:/language/ControlPanel/
```
Comment (Jermolene) [inline editions/es-ES/tiddlers/es-ES/ControlPanel.multids]: I'm afraid there's another issue here: these tiddlers are actual Spanish translation of the core strings. They should be in `languages/es-ES`, not here in the `es-ES` edition.

**Comment by Jermolene** [comment]
Hi @PauDeLuca one last problem, I'm afraid!

Some of the files you've got in `editions/es-ES` are actually the core translation files that should be in `languages/es-ES`. You'll need to move them to the correct folder.

**Comment by PauDeLuca** [comment]
I see the download-empty reminiscence. I'll correct it right now.

The updated language pack folder is already in languages/es-ES. However, it
doesn't load yet when starting TiddlyWiki. I've provisorily placed an
updated language folder in es-ES/tiddlers/ to force the loading while
working on the translation. It's intended to remove it once the translation
is finished.

I'm unsure about the file editions/fr-FR/tiddlers/$__StoryList.tid
<https://github.com/Jermolene/TiddlyWiki5/pull/2368#discussion_r59450224>.
As I see, is the story list created while searching for the templates
containing fr-title I needed to edit to allow es-title. Should I erase it?

Thank you very much

2016-04-12 22:50 GMT+02:00 Jeremy Ruston <notifications@github.com>:

**Pull request closed without merging**